src/Symfony/Component/Form/Extension/Core/DataTransformer/MoneyToLocalizedStringTransformer.php

```diff
@@ -68,6 +73,8 @@ public function transform($value)
     /**
      * Transforms a localized money string into a normalized format.
      *
+     * @requires extension bcmath
```

Comment:
I think this should be removed or replaced with [@uses](https://docs.phpdoc.org/references/phpdoc/tags/uses.html).

Reply:
it's not needed anymore